src/Components/homeComponent.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+import '../CSS/signupComponent.css';
+ 
+class homeComponent extends Component {
+    render() {
+        return (
+            <div><h1 className="home">Home</h1>
+                <img src={require('../images/website.jpg')} alt="image" className="homeimage" />
```

Comment:
Keep images in public folder
SOLUTION Refrence: https://stackoverflow.com/questions/47196800/reactjs-and-images-in-public-folder/48730272